src/Place/CommandHandler.php

```diff
@@ -199,6 +200,17 @@ protected function getFlagAsInappropriateClassName()
     }
 
     /**
+     * @param UpdateAddress $updateAddress
+     */
+    protected function handleUpdateAddress(UpdateAddress $updateAddress)
+    {
+        /* @var Place $place */
+        $place = $this->offerRepository->load($updateAddress->getItemId());
+        $place->updateAddress($updateAddress->getAddress(), $updateAddress->getLanguage());
+        $this->offerRepository->save($place);
+    }
```

Comment:
Eventueel een if statement inbouwen om enkel een event aan te maken als het adres is aangepast?